frontend/app/vite.config.ts

```diff
@@ -48,6 +48,12 @@ if (!hmrEnabled)
 
 const enableChecker = !(process.env.CI || isTest || process.env.VITEST);
 
+const builtInModulesToNotExternalized = [
+  'buffer',
+  'crypto',
+  'events',
+];
```

Comment:
Without this, I got this error after build:
```
Uncaught TypeError: Failed to resolve module specifier "buffer". Relative references must start with either "/", "./", or "../".
```

To be honest I'm not 100% sure why it happens.
But 1 things for sure, these 3 built in modules are imported in the `reown` package

Reply:
Weird, for nuxt in the website we didn't have to do anything similar right?

Reply:
probably because it's nuxt, there is likely a difference in the building process.